nimble/host/audio/services/bass/include/services/bass/ble_audio_svc_bass.h

```diff
@@ -0,0 +1,450 @@
+    BLE_SVC_AUDIO_BASS_BIG_ENC_BROADCAST_CODE_REQ,
+
+    /** BLE AUDIO BASS BIG Encryption: Decrypting */
+    BLE_SVC_AUDIO_BASS_BIG_ENC_DECRYPTING,
+
+    /** BLE AUDIO BASS BIG Encryption: Bad Code */
+    BLE_SVC_AUDIO_BASS_BIG_ENC_BAD_CODE
+};
+
+/** BLE AUDIO BASS PA Sync parameters, valid fo Modify Source operation */
+enum ble_svc_audio_bass_pa_sync {
+    /** BLE AUDIO BASS PA Sync: Do not synchronize to PA */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_DO_NOT_SYNC,
+
+    /** BLE AUDIO BASS PA Sync: Synchronize to PA – PAST available */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_SYNC_PAST_AVAILABLE,
+
+    /** BLE AUDIO BASS PA Sync: Synchronize to PA – PAST not available */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_SYNC_PAST_NOT_AVAILABLE,
+
+    /**
+     * BLE AUDIO BASS PA Sync: reserved for future use.
+     * This shall be always last value in this enum
+     */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_RFU
+};
+
+/** BLE AUDIO BASS Broadcast Receiver: PA Sync States */
+enum ble_svc_audio_bass_pa_sync_state {
+    /** BLE AUDIO BASS PA Sync State: Not synchronized to PA */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_NOT_SYNCED,
+
+    /** BLE AUDIO BASS PA Sync State: SyncInfo Request */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNC_INFO_REQ,
+
+    /** BLE AUDIO BASS PA Sync State: Synchronized to PA */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNCED,
+
+    /** BLE AUDIO BASS PA Sync State: Failed to synchronize to PAA */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNCED_FAILED,
+
+    /** BLE AUDIO BASS PA Sync State: No PAST */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_NO_PAST
+};
+
+/** BLE AUDIO BASS Broadcast Receiver State: Subgroup entry */
+struct ble_svc_audio_bass_subgroup {
+    /** BLE AUDIO BASS Subgroup entry: Bis Synchronization State */
+    uint32_t bis_sync_state;
+
+    /** BLE AUDIO BASS Subgroup entry: Metadata length */
+    uint8_t metadata_length;
+
+    /** BLE AUDIO BASS Subgroup entry: Metadata */
+    uint8_t *metadata;
+};
+
+/** BLE AUDIO BASS Broadcast Receiver State */
+struct ble_svc_audio_bass_receiver_state {
+    /** BLE AUDIO BASS Broadcast Receiver State: Source ID */
+    uint8_t source_id;
+
+    /** BLE AUDIO BASS Broadcast Receiver State: Source BLE Address */
+    ble_addr_t source_addr;
+
+    /** BLE AUDIO BASS Broadcast Receiver State: Source Advertising SID */
+    uint8_t source_adv_sid;
+
+    /** BLE AUDIO BASS Broadcast Receiver State: Broadcast ID */
+    uint32_t broadcast_id;
+
+    /** BLE AUDIO BASS Broadcast Receiver State: PA Sync state */
+    enum ble_svc_audio_bass_pa_sync_state pa_sync_state;
+
+    /** BLE AUDIO BASS Broadcast Receiver State: BIG Encryption */
+    enum ble_svc_audio_bass_big_enc big_encryption;
+
+    /**
+     * BLE AUDIO BASS Broadcast Receiver State: Bad Code.
+     * On GATT Read access, this value is ignored if big_encryption
+     * is not set to BLE_SVC_AUDIO_BASS_BIG_ENC_BAD_CODE
+     */
+    uint8_t bad_code[BLE_AUDIO_BROADCAST_CODE_SIZE];
```

Comment:
const?

Reply:
these are set from data received on control point - we cant make these const